CSharp/Samples/AlarmBot/Models/AlarmModule.cs

```diff
@@ -20,7 +20,7 @@ protected override void Load(ContainerBuilder builder)
         {
             base.Load(builder);
 
-            builder.Register(c => new LuisModelAttribute("c413b2ef-382c-45bd-8ff0-f76d60e2a821", "6d0966209c6e4f6b835ce34492f3e6d9")).AsSelf().SingleInstance();
+            builder.Register(c => new LuisModelAttribute("c413b2ef-382c-45bd-8ff0-f76d60e2a821", "6d0966209c6e4f6b835ce34492f3e6d9")).AsSelf().AsImplementedInterfaces().SingleInstance();
```

Comment:
How common is this pattern? Should we temporarily add a constructor overload that takes in a `LuisModelAttribute` and passes it to the main ctor, or does the extra complexity outweigh the benefit?